src/mumble/MainWindow.cpp

```diff
@@ -3350,6 +3355,15 @@ void MainWindow::on_gsHelpVersionCheck_triggered(bool down, QVariant) {
 	versionCheck();
 }
 
+void MainWindow::on_gsTogglePositionalAudio_triggered(bool down, QVariant) {
+	if (!down) {
+		return;
+	}
+
+	// WIP: Toggle positional audio
+	Global::get().pluginManager->togglePositionalAudio();
```

Comment:
I assume this is the part where you are unsure what to do. If I understand the original feature request correctly, we simply want a shortcut that toggles positional audio on a global level. That should mean it will be enough to toggle the global positional audio setting. (the same as toggling the positional audio checkbox in the respective settings page)

So lets look at the AudioOutput settings page UI definition:

https://github.com/mumble-voip/mumble/blob/c4b5858d141f76cce553be2f74dfc4291989fc9b/src/mumble/AudioOutput.ui#L489-L495

There we can see that the checkbox is named ``qcbPositional``

We can now scan the code for ``qcbPositional`` and find the respective Mumble setting here:

https://github.com/mumble-voip/mumble/blob/c4b5858d141f76cce553be2f74dfc4291989fc9b/src/mumble/AudioConfigDialog.cpp#L766

Which means we are interested in toggling ``bPositionalAudio``.

So the next step is to figure out how to do that. Luckily, other shortcuts exists which already toggle some settings. Lets look at "Toggle Text-To-Speech":

https://github.com/mumble-voip/mumble/blob/7bed82e8c2aba2d4405861673e2af688e1a5b819/src/mumble/MainWindow.cpp#L3326-L3332
https://github.com/mumble-voip/mumble/blob/7bed82e8c2aba2d4405861673e2af688e1a5b819/src/mumble/MainWindow.cpp#L4152-L4154

So I think it should be enough to follow the same steps as the ``Toggle Text-To-Speech`` shortcut, but with ``bPositionalAudio`` instead of ``bTTS``.

Reply:
Awesome, thank you so much! I'll get on that right away.